### ISSUE TITLE
JSON schema not being updated with plugin version changes

### ISSUE THREAD

**Issue by MariaSolOs** [dprint/dprint-vscode#46]
Today I bumped the [TypeScript plugin](https://github.com/dprint/dprint-plugin-typescript) to version 0.76.0, but when setting the `jsx.bracketPosition` setting (which is being added in this version) in my `dprint.json` file, I didn't get any JSON completions. The version in the plugin array is `0.76.0` and I can see the new setting being applied, so this seems like a schema issue.

The logs show that the extension is reading some cached files in Library/Caches/dprint/cache, and when I checked those out I noticed that the plugin version was still `0.75.0`. I tried removing the cache manifest but it didn't fix the issue.

Maybe I'm just too eager to try the new setting lol because I can also see that [the docs](https://dprint.dev/plugins/typescript/config/) haven't been updated yet.

**Comment by dsherret** [comment]
I looked into this just now and at the time I had accidentally not added "jsx.bracketPosition" as a definition. I did notice and fixed the schema in version 0.77.0 though here: https://github.com/dprint/dprint-plugin-typescript/commit/634ee00f6e9cdae14a896e5899afb04433d07b50#diff-7d120fa4e7f3ee14ec5018ae90bd1839c124f2942d960130c75cb63c2052ae30R796-R798

So yeah, this was just me messing up the schema. Sorry about that! Should be fixed as of a month ago with version 0.77.0.
